# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/jimmy/svgalib/gcc-3.3.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./demos/lineart.c.orig	2000-02-19 11:12:28.000000000 +0200
+++ ./demos/lineart.c	2003-10-19 08:40:01.000000000 +0300
@@ -19,8 +19,7 @@
   
   if(mode == 0)
     {
-      printf("Usage:linp [mode mode ...]\n
-              where mode is an integer.\n");
+      printf("Usage:linp [mode mode ...]\n\nwhere mode is an integer.\n");
       return;
     }
   if(! vga_hasmode(mode)) {diff -Nru3 trunk-1582/package/jimmy/svgalib/installmodule.patch trunk/package/jimmy/svgalib/installmodule.patch
